client/my-sites/plugins/plugins-browser/index.jsx

```diff
@@ -616,6 +657,8 @@ export default flow(
 				selectedSite: getSelectedSite( state ),
 				siteSlug: getSelectedSiteSlug( state ),
 				sites: getSelectedOrAllSitesJetpackCanManage( state ),
+				isRequestingRecommendedPlugins: getRecommendedPlugins( state, selectedSiteId ) === null,
```

Comment:
We seem to be calling `getRecommendedPlugins( state, selectedSiteId )` twice. Could we store it in a variable above?

Alternatively, we could remove `isRequestingRecommendedPlugins` completely, and use `recommendedPlugins === null` where necessary (totally optional and up to you).